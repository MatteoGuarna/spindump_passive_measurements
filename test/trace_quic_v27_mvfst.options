--report-packets --format text --report-notes
