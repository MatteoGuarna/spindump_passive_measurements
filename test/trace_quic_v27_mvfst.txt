simple test with mvfst in draft 27 mode
