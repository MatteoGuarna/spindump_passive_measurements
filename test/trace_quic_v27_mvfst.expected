QUIC 127.0.0.1 <-> 127.0.0.1 08756fbcb0ea7166-746fbcb0ea716665 (50392:9999) at 1585323473689766 new starting packets 0 0 bytes 0 0 note "No rsp,v27,no spin"
QUIC 127.0.0.1 <-> 127.0.0.1 08756fbcb0ea7166-08400000437c4033 (50392:9999) at 1585323473690300 change starting packets 1 0 bytes 1280 0 bandwidth 1280 0 note "No rsp,v27,no spin"
QUIC 127.0.0.1 <-> 127.0.0.1 08756fbcb0ea7166-08400000437c4033 (50392:9999) at 1585323473690300 measurement up right 534 packets 1 0 bytes 1280 0 bandwidth 1280 0 note "No rsp,v27,no spin"
QUIC 127.0.0.1 <-> 127.0.0.1 756fbcb0ea716665-08400000437c4033 (50392:9999) at 1585323473690394 packet up responder length 165 packets 1 2 bytes 1280 240 bandwidth 1280 240 note "V27,no spin"
QUIC 127.0.0.1 <-> 127.0.0.1 756fbcb0ea716665-08400000437c4033 (50392:9999) at 1585323473690412 packet up responder length 779 packets 1 3 bytes 1280 1019 bandwidth 1280 1019 note "V27,no spin"
QUIC 127.0.0.1 <-> 127.0.0.1 756fbcb0ea716665-08400000437c4033 (50392:9999) at 1585323473690609 packet up initiator length 75 packets 2 3 bytes 1355 1019 bandwidth 1355 1019 note "V27,no spin"
QUIC 127.0.0.1 <-> 127.0.0.1 756fbcb0ea716665-08400000437c4033 (50392:9999) at 1585323473691147 packet up initiator length 113 packets 3 3 bytes 1468 1019 bandwidth 1468 1019 note "V27,no spin"
QUIC 127.0.0.1 <-> 127.0.0.1 756fbcb0ea716665-08400000437c4033 (50392:9999) at 1585323473691215 packet up responder length 74 packets 3 4 bytes 1468 1093 bandwidth 1468 1093 note "V27,no spin"
